boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059.rst

```diff
@@ -135,6 +134,48 @@ Here is an example for the :ref:`blinky-sample` application.
 
 Observe the LED on the board blinking.
 
+Flashing with a device firmware update (DFU)
+============================================
+
+The board is factory-programmed with Nordic's bootloader from Nordic's nRF5 SDK.
+This section covers the steps required to migrate to MCUboot and use its serial
+recovery mode over USB CDC ACM to program new firmware.
+
+Obtain a Nordic-signed MCUboot firmware image, then perform a firmware update
+over USB using Nordic's nrfutil command line utility as follows::
+
+	nrfutil dfu usb_serial -pkg mcuboot-img.zip -p /dev/ttyACM0
+
+For more information on this step visit: `Nordic Semiconductor USB DFU`_ and
+`nrfutil GitHub`_ pages. At the end of this step, MCUboot will have replaced
+Nordic's bootloader in flash.
+
+Next, compile the application to be flashed on the device.
+Select :option: `CONFIG_BOOTLOADER_MCUBOOT`, under "Boot options" and set
+:option: `CONFIG_TEXT_SECTION_OFFSET` under "Build and Link features",
+"Linker options" to 0x200 to ensure the code is offset to account for MCUboot
+firwmare image metadata.
+
+Sign the resulting firmware image using the imgtool utility as follows::
+
+	imgtool sign -S 0x60000 --key mcuboot/root-rsa-2048.pem \
```

Comment:
from perspective of mcuboot root directory this command line should looks like this:
./scripts/imgtool.py sign -S 0x60000 --key root-rsa-2048.pem --header-size 0x200 --align 8 --version 3.0 firmware.bin signed_firmware.bin